test/run-test.ps1

```diff
@@ -42,7 +42,7 @@ pushd $repoRoot
 
 # Loop through each sdk Dockerfile in the repo and test the sdk and runtime images.
 Get-ChildItem -Recurse -Filter Dockerfile |
-    where DirectoryName -like "*${dirSeparator}${imageOs}${dirSeparator}sdk${dirSeparator}*" |
+    where DirectoryName -like "*${dirSeparator}${imageOs}${dirSeparator}sdk*" |
```

Comment:
For 1.0 and 1.1, the sdk directory always has children - msbuild and projectjson.  Since the 2.0 sdk doesn't have those variations, it is a leaf directory therefore doesn't end with a dirSeparator.  Because of this, the tests were not run against the 2.0 assets.